src/main/scala/viper/gobra/frontend/info/ExternalTypeInfo.scala

```diff
@@ -92,4 +92,8 @@ trait ExternalTypeInfo {
 
   /** returns the code root for a given node; can only be called on nodes that are enclosed in a code root */
   def codeRoot(n: PNode): PCodeRoot with PScope
+
+  def enclosedInLoop(n: PNode) : Option[PForStmt]
+
+  def enclosedInLabeledLoop(label: PLabelUse, n: PNode) : (Option[PForStmt], Vector[PExpression])
```

Comment:
Maybe a better name would be `enclosingLabeledLoop`. What is the second elemnt of the returned tuple here? I would add comments to both methods introduced in this file